src/scss/includes/component.scss

```diff
@@ -3,10 +3,10 @@
 $panel_width : 400px;
 
 /* text */
-$primary_text: #353c52;
+$primary_text: #0c0c0e;
```

Comment:
This is a change for the whole app, and quite big for this primary color going from gray to black. I don't think this is a problem, but it probably shouldn't have been in a PR related to a single feature.

Reply:
Yes, I missed to note this change in the PR's description. Do you think this could cause inconsistencies between expected design and the app ?

Should we Revert this change and open another PR focused on color changes ?

Reply:
I don't think it's a big deal, as long as it evolved toward the target design. But let's split such changes next time :)